README.md

```diff
@@ -13,6 +13,11 @@ $ sudo apt-get install libxml2 libxml2-dev
 
 Or build from source. You would need it installed to use ``gosax``.
 
+##improvements
```

Comment:
This is probably unnecessary, given that it's in the main repo now and not in a fork